### ISSUE TITLE
[BUG] Auto Module only uses module 0

### ISSUE THREAD

**Issue by SuperTux88** [ShadowTheAge/yafc#47]
- Latest version 0.4.7 on Linux
- Mods: Seablock with latest mod versions. I've put an empty save with the enabled mods to sync to [here](https://files.coding4.coffee/selif/seablock-empty-yafc-auto-modules.zip), that's probably easier and faster than packing my mods folder (and a lot smaller). However, as described in #46, I'm not sure if YAFC is really using these versions or older versions that are still in my mods folder.

Since I have modules in my game now, I wanted to test the Auto Module feature, but I'm a bit confused, with Bobs modules in Seablock, you have Modules 0 to 3 for each type. However the Auto Module always only puts the lowest tier module 0 into the machines, even when I set the slider all the way to the max "Use best modules", but I would have expected that module 3 are better.

How does it decide which modules are the best?

**Comment by ShadowTheAge** [comment]
It only puts modules that are accessible by your current milestones

**Comment by SuperTux88** [comment]
Oh, I just noticed that one milestone was disabled again which I thought I already had enabled (but maybe I didn't save?). I thought they were all accessible because I could select the modules manually without them being grayed out, but it didn't select them automatically. I enabled that milestone now (and saved so it hopefully stays enabled now) and now it selects module 3. Sorry for the false alarm.
